src/wasm/tests/BUILD

```diff
@@ -1,4 +1,8 @@
-load("//third_party/java/arcs/build_defs:build_defs.bzl", "arcs_ts_test")
+load(
+    "//third_party/java/arcs/build_defs:build_defs.bzl",
+    "arcs_serialize_manifest",
+    "arcs_ts_test",
```

Comment:
Not used?

Reply:
`arcs_ts_test` is used in this BUILD file.

Reply:
Oh whoops, I was only looking at the diff..